widgets/src/google/Common.js

```diff
@@ -21,11 +21,15 @@
     };
     Common.prototype = Object.create(HTMLWidget.prototype);
 
-    Common.prototype.publish("chartAreaWidth", "80%", "string", "Chart Area Width");
+    Common.prototype.publish("chartAreaWidth", "80%", "string", "Chart Area Width"); // num or string
     Common.prototype.publish("chartAreaHeight", "80%", "string", "Chart Area Height");
 
+    Common.prototype.publish("chartAreaTop", "auto", "string", "Chart Area Distance From Top"); // num or string
```

Comment:
The default should probably be null?  (then it will default to googles default)?